src/components/modals/CreateGroupsOrTypesModal.svelte

```diff
@@ -0,0 +1,210 @@
+              <p><i>The newly created group will be empty, though you can upload sources into it.</i></p>
+              <div class="timeline-editor-layer-filter">
+                <input
+                  bind:value={newTypeName}
+                  on:change={() => (newTypeError = null)}
+                  autocomplete="off"
+                  class="st-input w-100"
+                  name="filter-ee"
+                  placeholder={'New Derivation Group Name'}
+                />
+                <select
+                  bind:value={newTypeSourceType}
+                  on:change={() => (newTypeError = null)}
+                  name="newTypeSourceType"
+                  class="st-select"
+                  style="width: 200px"
+                >
+                  {#each $externalSourceTypes as sourceType}
+                    <option value={sourceType.name}>{sourceType.name}</option>
+                  {/each}
+                </select>
+                <button class="st-button w-10" type="submit" on:click|preventDefault={onCreateDerivationGroup}>
+                  Create
+                </button>
+              </div></TabPanel
+            >
+            <TabPanel>
+              <p style="padding-top:10px">Provide a name for the new external source type.</p>
+              <p><i>The newly created external source type will be empty, though you can upload sources into it.</i></p>
+              <div class="timeline-editor-layer-filter">
+                <input
+                  bind:value={newTypeName}
+                  on:change={() => (newTypeError = null)}
+                  autocomplete="off"
+                  class="st-input w-100"
+                  name="filter-ee"
+                  placeholder={'New External Source Type Name'}
+                />
+                <button class="st-button w-10" type="submit" on:click|preventDefault={onCreateExternalSourceType}>
+                  Create
+                </button>
+              </div></TabPanel
+            >
+            <TabPanel>
+              <p style="padding-top:10px">Provide a name for the new external event type.</p>
+              <p><i>The newly created external event type will be empty, though you can upload events into it.</i></p>
+              <div class="timeline-editor-layer-filter">
+                <input
+                  bind:value={newTypeName}
+                  on:change={() => (newTypeError = null)}
+                  autocomplete="off"
+                  class="st-input w-100"
+                  name="filter-ee"
+                  placeholder={'New External Event Type Name'}
+                />
+                <button class="st-button w-10" type="submit" on:click|preventDefault={onCreateExternalEventType}>
+                  Create
+                </button>
+              </div></TabPanel
+            >
+          </Tabs>
+        </div>
+        <div style="flex-direction: column;">
+          <p>
```

Comment:
Can we not use a `<p>` tag here? `<p>` tags cannot contain `<div>` tags, which is what `<AlertError>` essentially is.